### ISSUE TITLE
Support for SRT file detection

### ISSUE THREAD

**Issue by doggy8088** [google/magika#77]
[SRT files](https://mailchimp.com/resources/what-is-an-srt-file/) are a common file format used to create subtitles. Magika seems not able to detect *.srt file.

**Comment by devilkadabra69** [comment]
srt file is not listed in [supported files](https://github.com/google/magika/blob/main/docs/supported-content-types-list.md)

**Comment by reyammer** [comment]
Thanks! would you mind adding more details about the SRT file format? We now have a github issue template with the info that would be very helpful: https://github.com/google/magika/blob/main/.github/ISSUE_TEMPLATE/new_content_type_request.md

Thanks!

**Comment by devilkadabra69** [comment]
ok so i have to add more details into [new_content_type_request.md](https://github.com/google/magika/blob/main/.github/ISSUE_TEMPLATE/new_content_type_request.md) ? or i have to remake it from scratch ?

**Comment by reyammer** [comment]
@devilkadabra69 / @doggy8088  if you could create a new issue with the proper information (eg, where to find samples, context on the content type, etc.), it would be very useful for tracking. Once this is done, please close this one to avoid duplicates. Thank you!

**Comment by doggy8088** [comment]
Ok. I'll provide full context and examples in the coming days.

**Comment by reyammer** [comment]
See #176 for more context. Thanks @doggy8088!
